extra/utils.py

```diff
@@ -86,12 +77,14 @@ def persistent_load(self, pid):
 
   return MyPickle(fb0).load(), key_prelookup
 
-def load_single_weight(t:Tensor, myfile, shape, strides, dtype, mmap_allowed=False):
+def load_single_weight(t:Tensor, myfile, shape, strides, dtype, storage_offset, mmap_allowed=False):
   bytes_size = np.dtype(dtype).itemsize
   if t is None:
     myfile.seek(prod(shape) * bytes_size, 1)
     return
 
+  myfile.seek(storage_offset * bytes_size, 1)
```

Comment:
tldr it was not handling `storage_offset`

Reply:
Hmm, so this is still isn't totally right. mmap uses a different path that doesn't respect this.